snagsby/sources.py

```diff
@@ -63,7 +58,54 @@ def options(self):
 
     @property
     def region_name(self):
-        return self.options.get('region')
+        return self.options.get('region', 'us-west-2')
```

Comment:
We specifically don't default this to allow the all the normal ways of configuring the SDK to take over. I see the `?region` as being similar to passing `--region` as a cli flag to awscli. Otherwise we want the normal SDK configuration like the `AWS_DEFAULT_REGION` environment variable, or config files, to step in as the normally would.

That's why we have this in the s3 source, only overriding region if explicitly set:

```
        s3_opts = {
            'config': Config(signature_version='s3v4'),
        }

        if self.region_name:
            s3_opts['region_name'] = self.region_name

        s3 = boto3.resource('s3', **s3_opts)
```